packages/legend-application-query-bootstrap/scripts/setup.js

```diff
@@ -77,6 +77,10 @@ export const setup = (outputDir) => {
               url: 'https://legend.finos.org/resource/studio/documentation/shared.json',
               simple: true,
             },
+            {
```

Comment:
like said, ideally, let's merge https://github.com/finos/legend/pull/566 first then we can also add https://legend.finos.org/resource/studio/documentation/dsl-service.json or something like that here. We should add this to other setup.js files as well